src/TTextEdit.cpp

```diff
@@ -531,6 +532,119 @@ void TTextEdit::updateLastLine()
     update(r);
 }
 
+/**
+ * @brief TTextEdit::getNextGraphemeSize
+ * @param str
+ * @param startOffset
+ * @param nextChar [out] character unicode
+ * @return number of character of grapheme
+ */
+inline int TTextEdit::getNextGraphemeSize(const QString& str, int startOffset, /* out */ uint *character)
```

Comment:
I think this is too simplistic - you will get a more thorough solution with a [`QTextBoundaryFinder`](https://doc.qt.io/qt-5/qtextboundaryfinder.html) set to hunt for a `QTextBoundaryFinder::Grapheme` type boundary - it is what I use...! :neckbeard:

Reply:
thanks